snuba/datasets/storages/factory.py

```diff
@@ -56,6 +48,20 @@
 from snuba.datasets.storages.transactions_ro import storage as transactions_ro_storage
 from snuba.datasets.storages.transactions_v2 import storage as transactions_v2_storage
 
+gen_metrics_dists_bucket_storage = build_writable_storage(
+    StorageKey.GENERIC_METRICS_DISTRIBUTIONS_RAW
+)
+gen_metrics_dists_aggregate_storage = build_readonly_storage(
+    StorageKey.GENERIC_METRICS_DISTRIBUTIONS
+)
+generic_metrics_sets_bucket_storage = build_writable_storage(
```

Comment:
Weird that this one is `generic_` instead of `gen_`.

Reply:
True